makerdao/governance-portal-v2
# Feat: hide tiny MKR votes and add impact tags info modal

### Link to Shortcut ticket:
https://app.shortcut.com/dux-makerdao/story/1853/add-shortcut-overlay-to-impact-poll-tags
https://app.shortcut.com/dux-makerdao/story/1714/hide-tiny-mkr-votes

### What does this PR do?
- It adds an info icon next to the poll tags section that triggers a modal on click giving a one-line explanation of impact estimation and linking to the corresponding section of the manual.
- It adds checkboxes in the vote breakdown section for polls and executives that when clicked, filter the list of voters including or excluding those with an MKR weight lower than 0.05.

### Steps for testing:
- Go to a poll
- Click on the info icon next to the poll tags
- Check that the modal renders
- Click out of the modal to exit
- Scroll down to the Votes by Address section
- Toggle on/off the "show <0.05 MKR voters" checkbox
- Check that the list of voters gets updated including/excluding those with an MKR weight lower than 0.05.
- Go to an executive vote
- Toggle on/off the "show <0.05 MKR voters" checkbox in the Supporters section
- Check that the list of voters gets updated including/excluding those with an MKR weight lower than 0.05.

### Screenshots (if relevant):
<img width="483" alt="image" src="https://user-images.githubusercontent.com/22449631/211634461-f54090b1-b834-4c7c-88a2-364e658cbf9c.png">
<img width="550" alt="image" src="https://user-images.githubusercontent.com/22449631/211634510-59207092-353e-425b-9018-762984655e38.png">
<img width="966" alt="image" src="https://user-images.githubusercontent.com/22449631/211634577-1e5a0c86-4a0f-46e5-90de-86280140d6a3.png">
<img width="960" alt="image" src="https://user-images.githubusercontent.com/22449631/211634621-58204ac9-c344-4a6f-8857-bbd0d73197f5.png">
<img width="434" alt="image" src="https://user-images.githubusercontent.com/22449631/211634959-141dba2a-d5de-4728-8e9e-ded118df9c4f.png">

### Add a GIF:
<img width="300" src="https://media.tenor.com/bMnEV8_OEQoAAAAd/tedlasso-wagmi.gif">
